### ISSUE TITLE
provide support for tibbles/csv files read with readr::read_csv

### ISSUE THREAD

**Issue by fmichonneau** [hms-dbmi/UpSetR#80]
It would be nice to support directly data frames stored as tibbles. Currently it fails as follows (I got the same error with my own dataset):

```r
> movies <- read_csv2( system.file("extdata", "movies.csv", package = "UpSetR"))
Using ',' as decimal and '.' as grouping mark. Use read_delim() for more control.
Parsed with column specification:
cols(
  .default = col_integer(),
  Name = col_character(),
  AvgRating = col_number()
)
See spec(...) for full column specifications.
> upset(movies)
Error in sort.list(y) : 'x' must be atomic for 'sort.list'
Have you called 'sort' on a list?
```

**Comment by tinyheero** [comment]
+1 

Ran into the same issue on my end. Got around by using `as.data.frame()`, but there shouldn't be any reason why it can't take a tibble directly?

**Comment by PKI-Kent** [comment]
I think this can be fixed by changing [line 5 of `Helper.funcs.R`](https://github.com/hms-dbmi/UpSetR/blob/master/R/Helper.funcs.R#L5) to 
`column <- data[[i]]`

The problem is that tibbles don't automatically convert to a vector when subset to a single column.

You could also use 
`column <- data[, i, drop=TRUE]`
but that will only work with tibble >= 1.4.1 (see tidyverse/tibble#311)

**Comment by zx8754** [comment]
Duplicate of https://github.com/hms-dbmi/UpSetR/issues/71

**Comment by webbedfeet** [comment]
Looks like this was "solved" two years ago by @mariobecerra but never pulled, so the problem still persists in version 1.4.0

**Comment by mariobecerra** [comment]
Yes, and I saw several other pull requests that never got attended. You can take a look at my fork and try to use it. I would help you more but I haven't used this in a long time so I don't remember.
